Directory.Build.targets

```diff
@@ -31,4 +32,8 @@
     <PackageReference Include="Microsoft.DotNet.XUnitExtensions" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(UseTargetFrameworkPackage)' != 'false'">
+    <PackageReference Include="Microsoft.DotNet.Build.Tasks.TargetFramework" Version="$(MicrosoftDotNetBuildTasksTargetFrameworkVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
+  </ItemGroup>
```

Comment:
Given that this changes the SDK targets significantly and we had issues with it in the past (i.e. VS support), I would make this opt-in instead of opt-out.

Reply:
I assume making this opt-in means creating an additional property that when set to true, this code is evaluated. Who would set such property to true?

Reply:
Only the project that needs the platform TFMs should opt-in to using this. You can keep the existing property from L35 but condition on true instead.